lib/Doctrine/ORM/Query.php

```diff
@@ -279,7 +277,7 @@ private function parse()
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
      */
     protected function _doExecute()
```

Comment:
return type here should probably be added

Reply:
That one causes so many test failures :scream:

Reply:
So is it actually `mixed`?

Reply:
Indeed, it's not just the tests, the sql executor hierarchy has inconsistent return type declarations. `MultiTableDeleteExecutor` declares `int`, so does `SingleTableDeleteUpdateExecutor`.